hack/verify-generated-docs.sh

```diff
@@ -9,23 +9,33 @@ source "${OS_ROOT}/hack/common.sh"
 
 cd "${OS_ROOT}"
 
-GENERATED_DOCS_ROOT="${OS_ROOT}/docs/generated"
-_tmp="${OS_ROOT}/_tmp"
-TMP_GENERATED_DOCS_ROOT="${_tmp}/generated"
+echo "===== Verifying Generated Docs ====="
 
-mkdir -p "${_tmp}"
-cp -a "${GENERATED_DOCS_ROOT}/" "${_tmp}"
+GENERATED_DOCS_ROOT_REL="docs/generated"
+GENERATED_DOCS_ROOT="${OS_ROOT}/${GENERATED_DOCS_ROOT_REL}"
+TMP_GENERATED_DOCS_ROOT_REL="_tmp/verify-generated-docs"
+TMP_GENERATED_DOCS_ROOT="${OS_ROOT}/${TMP_GENERATED_DOCS_ROOT_REL}/${GENERATED_DOCS_ROOT_REL}"
 
-"${OS_ROOT}/hack/update-generated-docs.sh"
-echo "diffing ${GENERATED_DOCS_ROOT} against freshly generated docs"
+mkdir -p "${TMP_GENERATED_DOCS_ROOT}"
+
+echo "Generating fresh docs..."
+set +o errexit
```

Comment:
I suggest dropping `set +o errexit` before and `set -o errexit` after and instead writing the code as follows:

```
echo "Generating fresh docs..."
if ! output=`${OS_ROOT}/hack/update-generated-docs.sh ${TMP_GENERATED_DOCS_ROOT_REL} 2>&1`
then
	echo "FAILURE: Generation of fresh docs failed:"
	echo "$output"
fi
```